Cargo.toml

```diff
@@ -19,13 +20,40 @@ include = [
     "/LICENSE-APACHE",
 ]
 
+[features]
+default = ["nb", "linux-embedded-hal"]
+nightly = ["async"]
```

Comment:
Is the `nightly` feature necessary?

Reply:
No, it just seemed like a convention to have a `nightly` feature. Removed.